### ISSUE TITLE
Frontend WMS service research and implementation

### ISSUE THREAD

**Issue by op-hub** [amosproj/amos2024ss04-building-information-enhancer#190]
## Description
See #119
Tldr: Satellite images as WMS service from BGK (Germany) or opengeodata (Bavaria)   

## Acceptance criteria
* Documentation
* Implentation for wms service to display satellite images in FE

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
